plugins/inputs/gnmi/README.md

```diff
@@ -64,11 +64,23 @@ It has been optimized to support gNMI telemetry as produced by Cisco IOS XR (64-
 
     ## If suppression is enabled, send updates at least every X seconds anyway
     # heartbeat_interval = "60s"
+
+  # [[inputs.gnmi.subscription]]
+  #    name = "descr"
+  #    origin = "openconfig-interfaces"
+  #    path = "/interfaces/interface/state/description"
+  #    subscription_mode = "on_change"
+
+  #    ## If tag_only is set, the subscription in question will be utilized to maintain a map of
+  #    ## tags to apply to other measurements emitted by the plugin, by matching path keys
+  #    ## All fields from the tag-only subscription will be applied as tags to other readings,
+  #    ## in the format <name>_<fieldBase>.
+  #    tag_only = true
```

Comment:
This is good to go, but I would really like the config to be formatted correctly in the README and in the example config. Removing extra white space, and only double, not triple pound sign for comments.

I believe this is more in-line with what we the config currently uses:

```
    # [[inputs.gnmi.subscription]]
    # name = "descr"
    # origin = "openconfig-interfaces"
    # path = "/interfaces/interface/state/description"
    # subscription_mode = "on_change"

    ## If tag_only is set, the subscription in question will be utilized to maintain a map of
    ## tags to apply to other measurements emitted by the plugin, by matching path keys
    ## All fields from the tag-only subscription will be applied as tags to other readings,
    ## in the format <name>_<fieldBase>.
    # tag_only = true
```

Reply:
@bewing is this something you could do and then we could land this?

Reply:
@sspaink has resolved this, thank you!